commands/mrHelper.js

```diff
@@ -538,18 +540,30 @@ module.exports = {
             { name: 'alphabetical', value: 'alphabetical' },
             { name: 'level', value: 'level' },
         ))
-    ),
+    )
+    .addSubcommand((subcommand) => subcommand.setName('says').setDescription('Hear what the Hero of Orgrimmar has to say'))
+    .addSubcommand((subcommand) => subcommand.setName('shouts').setDescription('We all know which quote you\'re looking for...')),
     async execute(interaction) {
         const subCommand = interaction.options.getSubcommand();
         if (subCommand === 'help') {
             const content = '**Mythic Rating Helper** can provide information about completing Mythic+ dungeons at certain keystone levels and how that might impact a character\'s Mythic+ rating.\n\n'
-                + '- `/mr-helper simulate` will simulate a character running all Mythic+ dungeons at a single keystone level\n'
-                + '- `/mr-helper push` will tell which dungeons a character could run to slightly improve their Mythic+ rating\n'
-                + '- `/mr-helper goal` will provide a plan for dungeons a character can complete to reach a goal Mythic+ rating\n\n'
+                + '- `/gamon simulate` will simulate a character running all Mythic+ dungeons at a single keystone level\n'
+                + '- `/gamon push` will tell which dungeons a character could run to slightly improve their Mythic+ rating\n'
+                + '- `/gamon goal` will provide a plan for dungeons a character can complete to reach a goal Mythic+ rating\n'
+                + '- `/gamon says` will treat you to a nice quote from everyone\'s favorite Tauren.\n'
```

Comment:
Again we should clarify this means only the user will see it. I also think it's beneficial to have the clarifying text in both the `help` argument and in the flavor text for the argument itself (referencing line 544)

Reply:
Updated this to match the description text in the command more closely.